src/main/java/bio/terra/cli/serialization/userfacing/UFConfig.java

```diff
@@ -91,7 +91,8 @@ public static class Builder {
     private Logger.LogLevel fileLoggingLevel;
     private Logger.LogLevel consoleLoggingLevel;
     private String serverName;
-    private UUID workspaceId;
+    // "id" instead of "userFacingId" because user sees this with "terra config list"
```

Comment:
I'm confused here. I thought you were keeping both IDs in the config.

Reply:
I'm keeping both in `PDWorkspace`. `PD` is physical disk -- what we're storing in `context.json` file. We need `uuid` there because WSM APIs take `uuid`, not `userFacingId`.

`UF` in `UFConfig` refers to what users see when they run a CLI command. I'd like to not show `uuid` there.